scss/_variables.scss

```diff
@@ -916,8 +916,8 @@ $dropdown-link-active-bg:           $component-active-bg !default;
 
 $dropdown-link-disabled-color:      $gray-600 !default;
 
-$dropdown-item-padding-y:           .25rem !default;
-$dropdown-item-padding-x:           1.5rem !default;
+$dropdown-item-padding-y:           $spacer / 4 !default;
```

Comment:
Any reason not to use `map-get($spacers, 1)` here — and eventually for each `$spacer / 2` , `*.25`, `*.5` etc.?

Reply:
If the developer uses a different `$spacers` map where there is no `1` key, this won't work.

Reply:
Oh right! Thanks :)